Vagrantfile

```diff
@@ -0,0 +1,27 @@
+Vagrant.configure("2") do |config|
```

Comment:
Please don't commit the Vagrantfile. Usually it should not be in your application directory.